tst/EnergyPlus/unit/ScheduleManager.unit.cc

```diff
@@ -1326,3 +1327,127 @@ TEST_F(EnergyPlusFixture, Schedule_GetCurrentScheduleValue_DST_RampUp_NoLeap)
         EXPECT_EQ(8760.0, HourOfYear);
     }
 }
+
+TEST_F(EnergyPlusFixture, ScheduleFileDSTtoggleOptionTest)
+{
+    // P. Shrestha and N. Merket - February 2022
+
+    // Create this file with test data in it. This test will fail until you do that.
+
+    // scheduleFile object of class "path" created here in the next line is basically a long string that provides the full path to the file
+    fs::path scheduleFile = configured_source_directory() / "tst/EnergyPlus/unit/Resources/schedule_file1.csv";
```

Comment:
This test CSV file has a value of 1 for the first 24 hours and a 0 for all the rest of the 8736 hours in a year.